src/iop/temperature.c

```diff
@@ -194,7 +194,7 @@ const char *name()
 
 const char **description(struct dt_iop_module_t *self)
 {
-  return dt_iop_set_description(self, _("scale raw RGB channels to balance white and help demosaicing"),
+  return dt_iop_set_description(self, _("scale raw RGB channels to balance white and help demosaicing and highlights reconstruction"),
```

Comment:
The chained _and_s are not easy to read. If I understand correctly, there is a list for the purposes. The first element is a single item; the second is a list itself:
- to balance white
- to help demosaicing and highlight reconstruction (_help_ has 2 objects)

(I'd also use _highlight_ in the singular form.)

Maybe:
scale raw RGB channels to balance white, and help demosaicing and highlight reconstruction
or
scale raw RGB channels to balance white, help demosaicing + highlight reconstruction